WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -220,6 +220,14 @@ protected SendControlViewModel(WalletService walletService, string title)
 					}
 				});
 
+			this.WhenAnyValue(x => x.Address, x => x.CustomChangeAddress)
+				.ObserveOn(RxApp.MainThreadScheduler)
+				.Subscribe(_ =>
+				{
+					this.RaisePropertyChanged(nameof(Address));
```

Comment:
I imagines you would have done....

```
Property
{
     set
     {
           RaisePropertyChanged (Property);
           RaisePropertyChanged (Property1)
     }
```

No need to get reactive ui involved here, and also it seperates the implementation of the properties into several places.

Also with this in the setter you dont have the recursive issue that your were suspecting, because its not under subscription.

Reply:
This is bad practice according to https://github.com/reactiveui/ReactiveUI/issues/223#issuecomment-15381281

Reply:
I cant agree with that, its from 2013 and reactiveUI people have changed their opinions on a lot of things since then.

For example ObservableToPropertyHelper... it usually complicates our code.... remember we dont need a space shuttle to get to school. Reactive UI doesnt need to be used for everything.

Putting it in the setter means everyone knows exactly what is going on. ReactiveUI is doing all kinds of stuff under the hood (scheduling, etc)

Best to Keep It Simple ;)

Reply:
Ok In this case im convinced, however just because we read another developer calling bad practice it doesnt necessarily make it true!

Reply:
That is right, but I had the same convention. With Reactive you should only have a set and get at the property no further logic. Then describe all relations in one place with WhenAny and others. 
It is good that you mentioned it because now I had to look into this.
https://reactiveui.net/docs/handbook/view-models/#best-practices